packages/react-scripts/template/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We can delete this file, I'd like to keep this PR focused. If we have a good reason for adding this and the corresponding `paths.js` changes, we can propose that in a new PR.